src/ProjectOrigin.WalletSystem.Server/Database/Mapping/ApplicationBuilderExtension.cs

```diff
@@ -11,7 +11,7 @@ public static void ConfigureSqlMappers(this IApplicationBuilder app)
     {
         var algorithm = app.ApplicationServices.GetRequiredService<IHDAlgorithm>();
 
-        SqlMapper.AddTypeHandler(new HDPrivateKeyTypeHandler(algorithm));
-        SqlMapper.AddTypeHandler(new HDPublicKeyTypeHandler(algorithm));
+        SqlMapper.AddTypeHandler<IHDPrivateKey>(new HDPrivateKeyTypeHandler(algorithm));
+        SqlMapper.AddTypeHandler<IHDPublicKey>(new HDPublicKeyTypeHandler(algorithm));
```

Comment:
Generic arguments can be inferred. They are not needed.

Reply:
done